modular_skyrat/code/datums/components/crafting/recipes/recipes_weapon_and_ammo.dm

```diff
@@ -21,9 +50,74 @@
 				/obj/item/stock_parts/cell/high = 1)
 	tools = list(TOOL_WELDER, TOOL_SCREWDRIVER, TOOL_WIRECUTTER)
 	time = 45
+	category = CAT_CLOTHING
+
+/datum/crafting_recipe/hazardarmor
+	name = "hazard vest armor"
+	result =  /obj/item/clothing/suit/armor/hazard
+	reqs = list(/obj/item/stack/sheet/metal = 1,
+				/obj/item/clothing/suit/hazardvest = 1,
+				/obj/item/stack/wrapping_paper = 5)
+	time = 100
+	category = CAT_CLOTHING
+
```

Comment:
Way too cheap for such a powerful item. **This gives non-security 35 melee armour when stacked with sec/dura jumpsuit.**
**This is your armour block:**
	armor = list(melee = 25, bullet = 10, laser = 0, energy = 0, bomb = 5, bio = 0, rad = 0)
**This is the durathread winter coat armour block:**
	armor = list("melee" = 20, "bullet" = 8, "laser" = 15, "energy" = 8, "bomb" = 25, "bio" = 10, "rad" = 15, "fire" = 75, "acid" = 37)
**This is the durathread vest armour block**
	armor = list("melee" = 20, "bullet" = 10, "laser" = 30, "energy" = 5, "bomb" = 15, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 50)
**This is the standard security armour (coat/vest)**
	armor = list("melee" = 30, "bullet" = 30, "laser" = 30, "energy" = 10, "bomb" = 25, "bio" = 0, "rad" = 0, "fire" = 50, "acid" = 50)

At the moment, this completely removes the durathread meta. Durathread requires cotton to be mutated and a lot of time to mass-produce, it also require leather and/or cloth to make durathread items. This takes virtually nothing to craft, all the items are in extreme abundance on the station with no access restrictions to get them. Once stacked with durathread, this gives you a hefty 35 melee armour.

Rework options:
• 10 melee armour, 5 bullet.
• 15 melee armour, 5 bullet. More expensive crafting (cable coil+plasteel?)
• 20 melee armour, 10 bullet, slowdown. ('cuz heavy hurrdurr)
• 5 melee armour, 10 to 15 bullet armour. Makes it less useful against NT, more useful againt syndies/nukies. A bit OP still.

Reply:
I'm gonna take the route of more expensive crafting.

Reply:
Done. Made durathread more consistent as well.